projects/go-lib/src/lib/components/go-toast/go-toast.component.scss

```diff
@@ -15,6 +15,15 @@
     justify-content: flex-end;
     padding: .875rem 1.25rem;
   }
+
+  &--dark {
+    background-color: $theme-dark-bg;
+    color: $theme-dark-color;
+
+    .go-toast-content--dark {
```

Comment:
This should be `.go-toast-content`

Sorry, I guess I forgot to clarify that it no longer needs to be a modifier class. We will be directly changing the `go-toast-content` class.